site/source/docs/api_reference/emscripten.h.rst

```diff
@@ -1286,10 +1288,10 @@ IndexedDB
   :param perror: An out parameter that will be filled with a non-zero value if an error occurred.
 
 
-Asyncify functions
-==================
+Fastcomp Asyncify functions
```

Comment:
Why was this changed to say "Fastcomp"? AFAIK same applies to Asyncify transform on top of wasm backend?

Reply:
This PR refactored these docs, and now this part has the fastcomp funcs, while lower down there is "Upstream Asyncify".

(Or maybe I didn't understand what you meant by "same" in that sentence?)

Reply:
By "same" I mean that the description in the first sentence (both `-s ASYNCIFY=1` as well as link to Asyncify docs) seems generic enough, but only included in Fastcomp section now.

Also, aren't the functions that are currently included in "Fastcomp functions" - e.g. `emscripten_sleep` - available with upstream backend as well?

If they're, then the current structure seems confusing, as it seems to imply that these functions are Fastcomp-only, while others are upstream-only. I'd suggest turning the first section into "Asyncify functions" and then nesting the other one under it as "Upstream-only Asyncify functions".

Reply:
Thanks! Now I see what you mean. Yeah, `sleep` looks wrong - it's mentioned in fastcomp, as you say, but should only be in the shared section earlier. I also see sleep with yield is in the wrong place. I opened #9819 now for these issues.